inc/woocommerce/class-storefront-woocommerce-adjacent-products.php

```diff
@@ -83,7 +83,7 @@ public function get_product() {
 			$this->current_product = $post->ID;
 
 			// Try to get a valid product via `get_adjacent_post()`.
-			while ( $adjacent = $this->get_adjacent() ) {
+			while ( $adjacent = $this->get_adjacent() ) { // phpcs:ignore WordPress.CodeAnalysis.AssignmentInCondition.FoundInWhileCondition
```

Comment:
I'd rather we didn't just add an ignore comment here. We should fix this issue carefully in its own PR - no need to rush these warning fixes IMO.